src/constants/exporter-name.constant.ts

```diff
@@ -0,0 +1,3 @@
+export const EXPORTER_NAME = {
+  REGEX: /^[\w\-.()\s]+$/,
```

Comment:
Automatically changed from `/^[A-Za-z\d\-._()\s]+$/` by linter. `\w` - 
Matches any word character (alphanumeric & underscore). Only matches low-ascii characters (no accented or non-roman characters). Equivalent to [A-Za-z0-9_]